soc/xtensa/intel_adsp/ace/include/intel_ace15_mtpm/adsp_shim.h

```diff
@@ -49,7 +49,67 @@ struct cavs_shim {
 	uint32_t svcfg; /* Offset: 0xf4 */
 	uint32_t _unused9[2];
 };
+
+/**
+ * DfPMCCH
+ * Power Management / Clock Control (HST) Registers
+ *
+ * Offset: 0x00071AC0h + 4000000h * f //f = 0 - HFC - 1
+ * These registers block (HST domain) are for general power management
+ * and clock control operation for DSP FW.
+ */
+struct ace_dfpmcch {
+	uint32_t dfspsreq;	/* Offset: 0x00 */
+	uint32_t _unused0[3];
+	uint32_t dfspsrsp;	/* Offset: 0x10 */
+	uint32_t _unused1[1];
+	uint32_t svcfg;		/* Offset: 0x18 */
+	uint32_t dfltrc;	/* Offset: 0x1c */
+	uint32_t _unused2[8];
+};
+
+/**
+ * DfPMCCU
+ * Power Management / Clock Control (ULP) Registers
+ *
+ * Offset: 0x00071B00h + 4000000h * f //f = 0 - HFC - 1
+ * These registers block (ULP domain) are for general power management
+ * and clock control operation for DSP FW.
+ */
+struct ace_dfpmccu {
+	uint32_t dfpmccap;	/* Offset: 0x00 */
+	uint32_t dfhrosccf;	/* Offset: 0x04 */
+	uint32_t dfxosccf;	/* Offset: 0x08 */
+	uint32_t dflrosccf;	/* Offset: 0x0c */
+	uint32_t dfsiorosccf;	/* Offset: 0x10 */
+	uint32_t dfhsiorosccf;	/* Offset: 0x14 */
+	uint32_t dfipllrosccf;	/* Offset: 0x18 */
+	uint32_t dfirosccv;	/* Offset: 0x1c */
+	uint32_t dffbrcfd;	/* Offset: 0x20 */
+	uint32_t dfapllptr;	/* Offset: 0x24 */
+	uint32_t dfclkctl;	/* Offset: 0x78 */
```

Comment:
Are the addresses assigned correctly? Based on the comments you're missing some `unused` fields